packages/apps/plugins/plugin-kanban/src/components/KanbanCard.tsx

```diff
@@ -50,15 +67,7 @@ export const KanbanCardComponent: FC<{
           <DotsSixVertical className={getSize(5)} />
         </button>
         <div className='flex flex-col grow pt-1'>
-          {model && (
-            <MarkdownEditor
-              model={model}
-              placeholder={t('item title placeholder')}
-              slots={{
-                root: { className: mx(focusRing, 'p-1') },
-              }}
-            />
-          )}
+          <TextEditor doc={doc} extensions={extensions} className={mx(focusRing, 'p-1')} />
```

Comment:
why does kanban card use `TextEditor` vs editor card uses the `useTextEditor` hook?

Reply:
Changed.